### ISSUE TITLE
Recommendation: Maybe a more realistic version

### ISSUE THREAD

**Issue by cubelodyte** [mchristopher/PokemonGo-DesktopMap#631]
Has anyone considered simply programming to use your 1, true Pokemon Go login?
Something that would simply help you find the Pokemon that are actually in your Nearby list?

This shouldn't come across as a bot.
This wouldn't be cheating* since you're only doing what the original version of the game implied it would do.

_* - I'm a little put off by the wanton misuse of the word "cheating" that's been flowing lately. In my understanding, one can't cheat if there are no rules. Given that Pokemon Go comes with no viable instructions and Niantic eschews communication, how could anyone be aware of a rule to break?! 
That said, violation of ToS is another matter entirely. Also, one can hardly justify gathering all of the spawn points an times within a mile radius (or more) as legitimate solution to the broken functionality in Pokemon Go._

**Comment by Aerokii** [comment]
Sadly, the logins we're using aren't really the issue- at present we can't even understand what the Pokemon GO servers are saying, which means until we figure that out there is no tracking. Like, at all.

That being said, what you want is basically this app with gps that follows the phone, and a limited radius (2 or 3, I think.) Shouldn't be that terribly difficult.

**Comment by HUN73R13** [comment]
I used an Android app called "Compass for Pokemon GO", it used a Pokemon trainer account to login, yet it doesn't work anymore. so we can confirm even official accounts can't access the API after the recent changes. 

however a group of devs at Reddit made some progress, you can follow up here: https://www.reddit.com/r/pokemongodev/comments/4w1cvr/pokemongo_current_api_status/

also keep an eye on Mike's twitter @Icer5k for updates about this app.

**Comment by Madman8** [comment]
how does SkipLagged do it? Can we reverse engineer what they're up to?

**Comment by HUN73R13** [comment]
@Madman8 wow! I just tested their website, it's working in realtime!!!! how long have they been up?

**Comment by mattmendoza** [comment]
They use a different method.  Their map works by gathering data from various users, it doesnt work in areas where there aren't many skiplagged users.  This software gets data directly from PoGo servers which is why they need to crack the API.

**Comment by Overgame007** [comment]
Just for information, the reverse engineering is done and they are coding the new API.

**Comment by kebabas** [comment]
Good job guys, keep it up :+1:

**Comment by PokemonCindy** [comment]
The Donation is coming, when this map run again !

**Comment by nickadimouse** [comment]
Im glad this App will be back up and running again ..I do not have a car so this makes it easier for me when i see a rare or a pokemon i am looking for in my neighborhood i am not wasting time running about trying to find it while its time is ticking off ..So thank you for all the hard work your putting into this ..

**Comment by hunterd95** [comment]
@HUN73R13 What website are you referring to?

**Comment by TheFalseKing** [comment]
I only know of 2 other sites that rely on crowdsourcing. Sadly only one I know of is working and it's a royal pain to use because you can't simply remove all the ratatta and pidgey spam from the screen. To all of you working on getting this up and running. Keep up the good work and thank you.

**Comment by GolbeZz** [comment]
@hunterd95 he is refering to skiplagged .. https://skiplagged.com/pokemon/#40.777,-73.984,13 .. if you are in a high traffic area it seems to work fine for some. However it doesnt work for me because im in a low volume area. Apparently its user based, rather than pulling data from servers, so I am assuming its not as reliable. Havn't tested it myself.

**Comment by GolbeZz** [comment]
@hunterd95 "edit* Ah skiplagged gotcha. theyre down for maintenance right now.. dangit!"

its working, just hit exit on the pop up.. you will have to zoom out and go to your location. The search doesnt work.

**Comment by hunterd95** [comment]
@GolbeZz Ty, going to wait an hour and see how accurate the chicago map is. :P

**Comment by TheFalseKing** [comment]
@hunterd95  I'm using it in NY and it's accurate. Missed a Charizard (spawned then disappeared) but got a porygon shortly afterwards.
(sorry for being offtopic)

**Comment by Optimus-Prance** [comment]
@Overgame007 like as in the desktop map?! Its really going to be back sometime?

**Comment by pill0ws** [comment]
@cubelodyte 

In response to your original post, Niantic refers to the PTA Guidelines as their adpoted set of "Rules". WIthin those guidelines is a category for cheating that lists a handful of examples but ultimately says "Methods of cheating, unfortunately, are limited only by cheaters’ imaginations". This rhetoric basically means "If you think its cheating then its cheating" However what it really should say is "Methods of cheating, unfortunately, are limited only by the game developers imaginations". This would be more accurate to what they really mean in this section. The guy who was playing the game with a drone was not considered cheating by Niantic. They even made a statement saying that as long as the phone visited those sites then it wasnt really cheating, the only way he could operate a phone strapped to a drone from a central hub would be listed under the same section as  "using modified or unofficial software". However they gave this guy the thumbs up even though he sat his butt at home and used a drone to play. Sadly, within their imagination, scanning areas to see what is there is considered cheating to them, even if you are going to those places in person to capture the things there. I learned  so many streets in my city that I had never knew existed with this App, I could deliver pizza like a BOSS here now. Apparently they dont care if you sit on your ass and play with a drone using unofficial software but if you travel all over town and learn every street in your area using unofficial software you are cheating

**Comment by Madman8** [comment]
SkipLagged does work, like I am 30 minutes outside Atlanta and my map is empty... totally. But I had a meeting downtown and in the city, I found a Magnamite and a Ivysaur (sp?) using the map. BUT the filter didn't work so I had to zoom in REALLY tight to avoid the pidgins and rats. 

Honestly, even Pokevision wasn't as good as this app. F-everything but this desktop client. It was so dead on. It not only had my little shitty neighborhood, and lured pokestops, and a way to connect to my phone, notifications, and a server mode,  but also, since using it, I now know where to drive to in order to check for rare pokemon. For the last two days I have been driving through this really sketchy section 8 apartment complex that has helped fill out my pokedex.  Because the app always sent me there. 

I get people walking after my car or up to my car, people standing in the road to slow me down, all kinds of weird stuff, and I really hope this thing comes back to life so I don't have to drive through 15 times a day to check for mon, and instead go back to being targeted and picking and choosing my times.

I WILL DONATE to the brilliant and tireless people who are busting their balls (and/or vaginas) to put together such a helpful, clean, polished, user friendly and well presented tool that truly makes the experience of this game what it was.  I really feel indebted to you for the enjoyment I have had over the last week or two. THANK YOU!

**Comment by rjansen90** [comment]
As far as I know, skiplagged uses a similar scanning technique, but stores all the pokemon on their servers. This way, the next person scanning the same area can start by collecting the existing pokemon before using the pesky niantic api again. 

Their map is of course inferior to this apps map, but maybe a similar thing can be done here (using their servers :P). They have a web api, which returns a list of pokemon in an area (example: http://skiplagged.com/api/pokemon.php?address=central%20park,%20new%20york,%20ny). It's possible to approximate when they were last scanned by finding the pokemon(s) with the longest time remaining. This app could then do some additional scanning to fill in the gaps.

**Comment by MacDogg** [comment]
Thanks for mentioning skiplagged.  I just checked it out and there are some pokes around my area.  Funny thing is I just saw a grimer a mile away and I thought..."Finally I can get one now that all the scanners are down and everyone won't know it's there!".  I get there, along with a ton of others, and the game just locks up.  I exit and restart and I cannot even hit OK on the splash message, just locks up.  This is the 5th time I have seen a grimer and **EVERY TIME** the game locks up.  Still haven't got one.
